lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,37 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server_host: 0.0.0.0
+  dev_server_port: 8080
+  dev_server_https: false
```

Comment:
Me neither 😄  Alright, makes sense so lets move the dev server to development only.